quinn/src/lib.rs

```diff
@@ -546,6 +323,266 @@ impl EndpointInner {
             x.notify();
         }
     }
+
+    fn poll_recv(
```

Comment:
`poll_*` naming in Futures code almost universally refers to a function that returns `Poll<_, _>`. Maybe these should be named `drive_*`?

Reply:
Okay.